### PR TITLE
idb plainmessage gasprices

### PR DESCRIPTION
**Pull request by turbocrime**
cc @TalDerei

### CODE REVIEW

**Comment by TalDerei** [comment]
@turbocrime bumped idb to enable gas fees on web

**Review by TalDerei — APPROVED** [review]

File: packages/storage/src/indexed-db/index.ts [hunk 1]
```diff
@@ -388,12 +392,16 @@ export class IndexedDb implements IndexedDbInterface {
     return SwapRecord.fromJson(json);
   }
 
-  async getGasPrices(): Promise<GasPrices | undefined> {
-    return this.db.get('GAS_PRICES', 'gas_prices');
+  async getGasPrices(): Promise<PlainMessage<GasPrices> | undefined> {
+    return await this.db.get('GAS_PRICES', 'gas_prices');
```
Comment (grod220) [inline packages/storage/src/indexed-db/index.ts:396]: Can we serialize/deserialize through json here?

Comment (grod220) [reply packages/storage/src/indexed-db/index.ts:396]: Also, think `GasPrices` type is better to passing around the application

File: packages/types/src/indexed-db.ts [hunk 2]
```diff
@@ -194,7 +195,7 @@ export interface PenumbraDb extends DBSchema {
   };
   GAS_PRICES: {
     key: 'gas_prices';
-    value: GasPrices;
+    value: PlainMessage<GasPrices>;
```
Comment (grod220) [inline packages/types/src/indexed-db.ts:198]: Let's make this `Jsonified<GasPrices>`

Comment (turbocrime) [reply packages/types/src/indexed-db.ts:198]: `PlainMessage` should provide better type safety than the de-typed output of `Jsonified` which is simply `JsonObject`

File: packages/storage/src/indexed-db/index.ts (hunk 1 above)
Comment (TalDerei) [reply packages/storage/src/indexed-db/index.ts:396]: this seems to just work finely

**Comment by TalDerei** [comment]
For context and visibility, the flow is:

1. Bump IDB version, clearing the database on upgrade,
2. Manually set default gas prices on indexDB initialization,
3. `initGasPrices` will only be called once on initialization, and condition will fail on subsequent extension startups,
4. Once non-zero gas fees are enabled in core, a single compact block's pb message will have a `gasPrices` field that won't be undefined,
5. In the block processor, it will call `saveGasPrices` and override the default value in indexDB,
6. remove the `initGasPrices` call

**Review by grod220 — APPROVED** [review]

**Pull request merged**